server/src/main/java/org/opensearch/common/cache/store/enums/CacheStoreType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.cache.store.enums;
+
+/**
+ * Cache store types in tiered cache.
+ *
+ * @opensearch.internal
+ */
+public enum CacheStoreType {
```

Comment:
Should we call this `CacheTieredStore` to make it more explicit

Reply:
Yes that was the reason to suggest the name as `CacheStoreType`